src/Consts.cs

```diff
@@ -87,6 +87,10 @@ public enum ReceiveType
         QueryNotice,
         CtcpReply,
         CtcpRequest,
+        CapLs,
+        CapList,
+        CapAck,
+        CapNak,
```

Comment:
Yes but it was the neatest way to do it. If you want me to move the regex checks into _Event_CAP and skip the use of ReceiveType I can do that, but it will be inconsistent with the rest of the code.